### PR TITLE
ENH Added point_size option for voronoi_plot_2d

### PR DESCRIPTION
**Pull request by mknorps**
When creating a plot with voronoi_plot_2d a default size of points was too big and there was no possibility to change that. I added a small enhancement to enable a point size change.

### CODE REVIEW

**Review by tylerjereddy — COMMENTED** [review]
This is probably a useful feature. The one broader concern I have is that we might eventually end up having to support a rather large number of keywords as the desire to customize these plots continues to increase (design concern re: supporting most keywords matplotlib allows).

I think when I really want to modify these plots I'll provide my own axis object to the `ax` argument, which gives the end user a fair bit more power in that regard.

So, I think I'm +1 here, but just want to make sure we carefully balance the desire for customization with the kw argument / support bloat concern.

**Comment by mknorps** [comment]
@tylerjereddy if you think it would be better to create the more customizable version of **voronoi_plot_2d** with ax objects I can work on that, but then it would be also a cool feature to have possibility to color polygons depending on their size.

I did this small enhancement, because when I created a Voronoi plot with >3000 point it was unreadable because of too big points. So this enhancement is mainly for readability and not something fancy.

I would appreciate suggestion on the further enhancement of this function.

File: scipy/spatial/_plotutils.py [hunk 1]
```diff
@@ -157,7 +160,8 @@ def voronoi_plot_2d(vor, ax=None, **kw):
         raise ValueError("Voronoi diagram is not 2-D")
 
     if kw.get('show_points', True):
-        ax.plot(vor.points[:,0], vor.points[:,1], '.')
+        point_size = kw.get('point_size', 2.0)
```
Comment (perimosocordiae) [inline scipy/spatial/_plotutils.py]: Default should be `None`, for consistency with user-specific matplotlib defaults.

**Review by perimosocordiae — COMMENTED** [review]
I also think that a more general/cleaner API is possible here, but I'm not opposed to merging this in the meantime.

File: scipy/spatial/_plotutils.py (hunk 1 above)
Comment (mknorps) [reply scipy/spatial/_plotutils.py]: Updated

**Review by perimosocordiae — APPROVED** [review]

**Comment by tylerjereddy** [comment]
Ok, in it goes, thanks @mknorps.

**Comment by endolith** [comment]
@tylerjereddy 

> The one broader concern I have is that we might eventually end up having to support a rather large number of keywords as the desire to customize these plots continues to increase 

There should be an option to change line styles, too.  :D

**Pull request merged**